nixos/roles/router/keepalived/dev.conf

```diff
@@ -68,7 +68,6 @@ vrrp_instance mgm {
     }
 
     virtual_ipaddress_excluded {
-        2a02:238:f030:1c1::1/64 preferred_lft 0
         172.20.1.1/24
```

Comment:
why is this going away instead of being replaced?

Reply:
It's already been replaced previously, see the address in the `virtual_ipaddress` block (not `_excluded`) for the mgm interface.